packages/crayons-icon/icon-utils/icon-registry.js

```diff
@@ -0,0 +1,12 @@
+/*global globalThis*/
+export const registerIconLibrary = (items) => {
+  globalThis.iconlibs = items;
```

Comment:
@detiwari003 rather than using the window reference why not make this utility part of the crayons-core? 

just like the `requestCache` reference, you can simply map to an internal reference within crayon's scope right!

Reply:
@detiwari003 Please test this in a simple HTML apps via cdn import as well.